qbox.sql

```diff
@@ -218,14 +218,35 @@ CREATE TABLE IF NOT EXISTS `playerskins` (
 CREATE TABLE IF NOT EXISTS `player_outfits` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `citizenid` varchar(50) DEFAULT NULL,
```

Comment:
FK?

Reply:
It's just a normal key. The name includes FK because it previously had a foreign key constraint but later removed due to it requiring changes in a lot of other scripts

Reply:
It shouldn't affect anything

Reply:
I'm wondering why citizenid is not a FK to the players table.